rocAL/rocAL/include/master_graph.h

```diff
@@ -145,6 +146,13 @@ class MasterGraph
     bool _output_routine_finished_processing = false;
     const RaliTensorDataType _out_data_type;
     bool _is_random_bbox_crop = false;
+    //box encoder
+    bool is_box_encoder = false;
```

Comment:
change the name to follow convention

Reply:
Done

Reply:
Done

Reply:
@swetha097 : I am not seeing the commit for this change